generator/src/lib.rs

```diff
@@ -0,0 +1,431 @@
+// pest. The Elegant Parser
+// Copyright (c) 2018 Dragoș Tiselice
+//
+// Licensed under the Apache License, Version 2.0
+// <LICENSE-APACHE or http://www.apache.org/licenses/LICENSE-2.0> or the MIT
+// license <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. All files in the project carrying such notice may not be copied,
+// modified, or distributed except according to those terms.
+
+//! # pest. The Elegant Parser
```

Comment:
I wouldn't add this documentation here. Can be empty until we get to document all internal crates.

Reply:
Alright, I've removed it for now, it was just duplicated from the derive crate.